server/sonar-web/pom.xml

```diff
@@ -230,7 +230,7 @@
         <dependency>
           <groupId>com.sonarsource</groupId>
           <artifactId>sonarsource-branding</artifactId>
-          <version>1.3.0.304</version>
+          <version>1.3.0.307</version>
```

Comment:
Ah ok, and it's because you updated the logo on sonarsource-branding. Got it, thanks!